samples/operate-first/Makefile

```diff
@@ -17,6 +19,8 @@ clean: clean
 .PHONY: helm-template
 helm-template: $(TOOLBIN)/yq $(TOOLBIN)/helm
 	$(TOOLBIN)/helm template $(CRD_RELEASE_NAME) -n $(NAMESPACE) $(CRD_CHART_PATH) > fybrik-crd.yaml
```

Comment:
@eletonia can you please add --set "vault.injector.enabled=false"   --set "vault.server.dev.enabled=true" same as in the quick start.

Reply:
@eletonia please rebase the pr as https://github.com/fybrik/fybrik/commit/80d2b8da84246560dbb9aea25ea781f42936cff5 contains changes to avoid using clusterRole if sclusterScope is specified. (we still need clusterRoleBinding for vault to create tokenreview resource though)